src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -42,6 +43,7 @@ class BasePaymentsPage extends React.Component {
             shouldShowAddPaymentMenu: false,
             shouldShowDefaultDeleteMenu: false,
             shouldShowPasswordPrompt: false,
+            shouldShowLoadingSpinner: true,
```

Comment:
Maybe we could initialize this to `false` so that when the component mounts we can render nothing if the data is fetched in less than the debounce time?